src/stacks-hierarchy/Root_PurchaseConfirmationScreen/Root_PurchaseConfirmationScreen.tsx

```diff
@@ -461,11 +417,11 @@ export const Root_PurchaseConfirmationScreen: React.FC<Props> = ({
                       'Ticketing',
                       'Pay with previous payment method clicked',
                       {
-                        paymentMethod: previousMethod?.paymentType,
+                        paymentMethod: previousPaymentOption?.paymentType,
```

Comment:
I think this line shows why it is easy to get confused here.
`method`, `option` and `type` - what is the current relationship between these terms, and what should it be? 🤔